packages/search-ui/src/SearchDriver.js

```diff
@@ -274,87 +278,91 @@ export default class SearchDriver {
    * Application state updates are performed in _updateSearchResults, but we
    * wait to make the actual API calls until all actions have been called.
    */
```

Comment:
Actually if there is on request I could make - it'd be great to add some `@params` docblocks here to more fully document the new option, e.g.

```
* @param {int} debounceInterval - ...etc
* @param {obj} options
* @param {bool} options.skipPushToUrl - Skips updating both the URL bar and browser history
* @param {bool} options.replaceUrl - Updates the URL bar, but does not insert an entry in the browser history
```

Something like that would probably help a lot with a new developer quickly grokking the difference between the two flags :)

Reply:
Great points.

Reply:
Updated here: https://github.com/elastic/search-ui/pull/449/commits/f60d07f3e241b922903339ad05610fc7e35ff305